docs/getting_started.md

```diff
@@ -104,63 +149,55 @@ class CNN(nn.Module):
     return x
 ```
 
-+++ {"id": "xDEoAprU6_JZ"}
++++ {"id": "hy7iRu7_zlx-"}
 
-## 3. Define loss
-
-We simply use `optax.softmax_cross_entropy()`. Note that this function expects both `logits` and `labels` to have shape `[batch, num_classes]`. Since the labels will be read from TFDS as integer values, we first need to convert them to a onehot encoding.
+### View model layers
 
-Our function returns a simple scalar value ready for optimization, so we first take the mean of the vector shaped `[batch]` returned by Optax's loss function.
+Create an instance of the Module and use the [`Module.tabulate`](https://flax.readthedocs.io/en/latest/api_reference/flax.linen.html#flax.linen.Module.tabulate) method to visualize a table of the model layers by passing an RNG key and template image input.
 
 ```{code-cell}
-:id: Zcb_ebU87G7s
-
-def cross_entropy_loss(*, logits, labels):
-  labels_onehot = jax.nn.one_hot(labels, num_classes=10)
-  return optax.softmax_cross_entropy(logits=logits, labels=labels_onehot).mean()
+---
+executionInfo:
+  elapsed: 100
+  status: ok
+  timestamp: 1671500846720
+  user:
+    displayName: Marcus Chiam
+    userId: '17531616275590396120'
+  user_tz: 300
+id: lDHfog81zLQa
+outputId: db2822ca-c9a4-4dec-c67b-a6788933bb2f
+---
+cnn = CNN()
+print(cnn.tabulate(jax.random.PRNGKey(0), jnp.ones((1, 28, 28, 1))))
 ```
 
-+++ {"id": "INZE3eM67JUr"}
-
-## 4. Metric computation
++++ {"id": "db132446"}
 
-For loss and accuracy metrics, create a separate function:
+## 5. Define loss
 
-```{code-cell}
-:id: KvuEA8Tw-MYa
-
-def compute_metrics(*, logits, labels):
-  loss = cross_entropy_loss(logits=logits, labels=labels)
-  accuracy = jnp.mean(jnp.argmax(logits, -1) == labels)
-  metrics = {
-      'loss': loss,
-      'accuracy': accuracy,
-  }
-  return metrics
-```
-
-+++ {"id": "lYz0Emry-ele"}
-
-## 5. Loading data
+We simply use `optax.softmax_cross_entropy()`. Note that this function expects both `logits` and `labels` to have shape `[batch, num_classes]`. Since the labels will be read from TFDS as integer values, we first need to convert them to a onehot encoding.
```

Comment:
Nit:
- Add a link to the Optax softmax cross entropy API doc and mention Optax since it's an external library.
- If you can, use "second person" like "you"/"your" (Google Style Guide).

For example:

```suggestion
For your loss, use a predefined [`optax.softmax_cross_entropy()`](https://optax.readthedocs.io/en/latest/api.html#optax.softmax_cross_entropy) from the Optax library. Note that this function expects both `logits` and `labels` to have shape `[batch, num_classes]`. Since the labels will be read from TFDS as integer values, first convert them to a one-hot encoding.
```